composer.json

```diff
@@ -11,7 +11,7 @@
 	"require": {
```

Comment:
I would suggest to use a require for "php 7.4" as the code is basically compatible with 7.4

Reply:
Hi @blacktek, unfortunately it is not backwards compatible with PHP 7.4 due to keywords such `mixed` being used as parameters or return types amongst other things.

Reply:
I'm currently using it on 7.4 too and is working. 7.4 doesn't support multiple union return types (e.g. string | bool), that I remember were not used. Are you sure that those are currently used?

Reply:
I assumed there would be problems since the psalm test fails but if these warnings/errors can be fixed quickly in order to return composer.json to 7.4 it may be worth it. I will look into it when I have some time.

Reply:
If you are struggling to set up my fork in `composer.json` here is an extract you can use:

```json
{
  "require": {
    "php": "^8.1",
    "vanilla/garden-cli": "dev-fix/php81"
  },
  "repositories": [
    { "type": "vcs", "url":  "https://github.com/donatello-za/garden-cli" }
  ]
}
```

Reply:
tnx a lot :) this is exactly what I was using.